src/main/java/com/ericsson/ei/subscription/SubscriptionRepeatDbHandler.java

```diff
@@ -82,70 +86,72 @@
      * Function that stores the matched aggregatedObjectId to the database.
      *
      */
-    public void addMatchedAggrObjToSubscriptionId(String subscriptionId, int requirementId, String aggrObjId)
-            throws Exception {
-
-        LOGGER.debug("Adding/Updating matched AggrObjId: " + aggrObjId + " to SubscriptionsId: " + subscriptionId
-                + " aggrId matched list");
+    public void addMatchedAggrObjToSubscriptionId(String subscriptionId, int requirementId, String aggrObjId) {
+        try {
+            LOGGER.debug("Adding/Updating matched AggrObjId: " + aggrObjId + " to SubscriptionsId: " + subscriptionId
+                    + " aggrId matched list");
 
-        if (checkIfAggrObjIdExistInSubscriptionAggrIdsMatchedList(subscriptionId, requirementId, aggrObjId)) {
-            LOGGER.debug("Subscription: " + subscriptionId + " and AggrObjId, " + aggrObjId
-                    + " has already been matched." + "No need to register the subscription match.");
-            return;
-        }
+            if (checkIfAggrObjIdExistInSubscriptionAggrIdsMatchedList(subscriptionId, requirementId, aggrObjId)) {
+                LOGGER.debug("Subscription: " + subscriptionId + " and AggrObjId, " + aggrObjId
+                        + " has already been matched." + "No need to register the subscription match.");
+                return;
+            }
 
-        if (!updateExistingMatchedSubscriptionWithAggrObjId(subscriptionId, requirementId, aggrObjId)) {
-            LOGGER.error("Couldn't update SubscriptionMatched id.");
-            LOGGER.debug(
-                    "New Subscription AggrId has not matched, inserting new SubscriptionId and AggrObjId to matched list.");
-            BasicDBObject document = new BasicDBObject();
-            document.put("subscriptionId", subscriptionId);
+            if (!updateExistingMatchedSubscriptionWithAggrObjId(subscriptionId, requirementId, aggrObjId)) {
+                LOGGER.error("Couldn't update SubscriptionMatched id.");
```

Comment:
Merge these two log messages to one log message.

Reply:
The first one seems redundant since the nested method also logs pretty muich the same thing, so I'll remove the first log message from here.